Sources/ClientRuntime/Networking/Http/HttpBody.swift

```diff
@@ -58,6 +58,7 @@ public extension HttpBody {
         }
     }
 
+    @available(*, deprecated, message: "This method is deprecated and will soon be removed. Call `readData()` instead.")
     func toData() throws -> Data? {
```

Comment:
The deprecation of this method is part of the reason you see so many new `await` calls in this PR.